CONTRIBUTING.md

```diff
@@ -22,7 +23,11 @@ Gonum discussion takes place on the [gonum-dev google group](https://groups.goog
 
 ### Code of Conduct
 
-In general follow the [Go code of conduct](https://golang.org/conduct).  If you are a contributor, please be welcoming to new contributors.  [Here](http://sarah.thesharps.us/2014/09/01/the-gentle-art-of-patch-review/) is a good guide.  If you are a new contributor, thank you!
+We expect the same values and behavior as described in [Go code of conduct](https://golang.org/conduct).
```

Comment:
I would prefer something like the following:

\```
When contributing or otherwise participating, please:

- Be friendly and welcoming
- Be patient
- Be thoughtful
- Be respectful
- Be charitable
- Avoid destructive behavior

Excerpted from the [Go conduct document](https://golang.org/conduct).
\```